### PR TITLE
chore: update webdriverio to v8

### PR DESCRIPTION
**Pull request by nolanlawson**
## Details

Updates WebDriverio to [v8](https://github.com/webdriverio/webdriverio/releases/tag/v8.0.0), and fixes some minor breaking changes in our tests.

## Does this pull request introduce a breaking change?

* ✅ No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

* ✅ No, it does not introduce an observable change.

### CODE REVIEW

File: packages/@lwc/integration-tests/scripts/wdio.conf.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const wdSuites = suites.reduce((seed, suite) => {
 exports.config = {
     logLevel: 'warn',
 
-    specs: ['./src/**/*.spec.js'],
+    specs: ['../src/**/*.spec.js'],
```
Comment (nolanlawson) [inline packages/@lwc/integration-tests/scripts/wdio.conf.js:50]: Now relative to the config file

File: packages/@lwc/integration-tests/scripts/commands/focus.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ module.exports = function (browser) {
     browser.addCommand(
         'focus',
         function () {
-            return this.execute(function (target) {
+            return browser.execute(function (target) {
```
Comment (nolanlawson) [inline packages/@lwc/integration-tests/scripts/commands/focus.js:20]: I think this just happened to work – `execute` should be on the `browser`, not `this` (which is an element).

**Comment by nolanlawson** [comment]
/nucleus ignore --reason 'downstream is failing for unrelated reason'

**Review by ravijayaramappa — APPROVED** [review]

**Review by ekashida — APPROVED** [review]

**Comment by nolanlawson** [comment]
/nucleus ignore --reason 'downstream is failing due to flapper'

**Pull request merged**